assets/js/romo/base.js

```diff
@@ -478,49 +469,46 @@ Romo.prototype.on = function(elem, eventName, fn) {
   }
   proxyFn._romofid = this._fn(fn)._romofid;
 
-  elem.addEventListener(eventName, proxyFn);
-  var key = this._handlerKey(elem, eventName, proxyFn);
-  if (this._handlers[key] === undefined) {
-    this._handlers[key] = [];
-  }
-  this._handlers[key].push(proxyFn);
-}
+  elems.forEach(Romo.proxy(function(elem) {
 
-Romo.prototype.off = function(elem, eventName, fn) {
-  var elemString = Object.prototype.toString.call(elem);
-  if (
-    elemString === '[object NodeList]'       ||
-    elemString === '[object HTMLCollection]' ||
-    Array.isArray(elem)
-  ) {
-    throw new Error('Can only unbind events on individual elems, not collections.');
-  }
+    elem.addEventListener(eventName, proxyFn);
 
-  var key = this._handlerKey(elem, eventName, fn);
-  (this._handlers[key] || []).forEach(function(proxyFn) {
-    elem.removeEventListener(eventName, proxyFn);
-  });
-  this._handlers[key] = [];
+    var key = this._handlerKey(elem, eventName, proxyFn);
+    if (this._handlers[key] === undefined) {
+      this._handlers[key] = [];
+    }
+    // `addEventListener` doesn't register duplicates (same function on same
+    // elem) so we shouldn't store them in our handlers arrays either
+    if (this._handlers[key].indexOf(proxyFn) === -1) {
+      this._handlers[key].push(proxyFn);
```

Comment:
@jcredding if duplicates aren't registered, why are we still pushing on to an array here?  Seems like we should just store one proxyFn per key (since the proxyFn is part of the key).

Reply:
@kellyredding - It's exact duplicates so if you did:

```javascript
Romo.on([window, window], 'click', someFunc)
```

This would only register one event handler. If you call `Romo.on` twice though like:

```javascript
Romo.on(window, 'click', someFunc)
Romo.on(window, 'click', someFunc)
```

This would register two events listeners because they build different `proxyFn` in the `on` method. This is what happens when we have multiple dropdowns open on a page (via a select in a dropdown or something). Since they are separate `Romo.on` calls they register two different `proxyFn` functions that end up calling the same romo dropdown function.